playbooks/storage_domain_workflow/create_storage_domain.yaml

```diff
@@ -0,0 +1,16 @@
+# Playbook to Create a storage domain (view-box).
+---
+- hosts: localhost
+  become: false
+  collections:
+    - cohesity.dataprotect
+  tasks:
+    - name: Create view box in the cohesity server.
+      cohesity_storage_domain:
+        cluster: "{{ cohesity_server }}"
+        username: "{{ cohesity_username }}"
+        password: "{{ cohesity_password }}"
+        validate_certs: "{{ cohesity_validate_certs }}"
+        name: 'Ansible'
+        state: 'present'
+        cluster_partition_name: "DefaultPartition"
```

Comment:
Question still remains, how does one get to know the partition name, if its is constant, can we just hard code it ?